docs/en/sql-reference/statements/check-table.md

```diff
@@ -10,9 +10,10 @@ The `CHECK TABLE` query in ClickHouse is used to perform a validation check on a
 
 Particularly it compares actual file sizes with the expected values which are stored on the server. If the file sizes do not match the stored values, it means the data is corrupted. This can be caused, for example, by a system crash during query execution.
 
-:::note
+:::warning
 The `CHECK TABLE`` query may read all the data in the table and hold some resources, making it resource-intensive.
 Consider the potential impact on performance and resource utilization before executing this query.
+This query will not improve performance of the system and you should not execute it if you are not sure of what you are doing.
```

Comment:
nit: the impact on performance was already mentioned twice here, maybe just add this:
```
You should not execute it if you are not sure of what you are doing.
```

Reply:
Yes, but some people still thought that running it would make queries faster once `CHECK TABLE` finishes